modules/installation-osp-dpdk-selecting-flavors.adoc

```diff
@@ -0,0 +1,45 @@
+:_content-type: PROCEDURE
+[id="installation-osp-dpdk-selecting-flavors_{context}"]
+= Selecting the huge pages size and CPU policy for virtual machines on OVS-DPDK hosts
+
+Each virtual machine that runs on a host that uses Open vSwitch with the Data Plane Development Kit (OVS-DPDK) requires huge pages and a dedicated CPU policy. Before you install a cluster that uses OVS-DPDK, create an {rh-openstack-first} flavor that meets this requirement.
+
+.Prerequisites
+
+* Check with your {rh-openstack} administrator to determine the maximum size and number of huge pages to use for each node.
+
+.Procedure
+
+* Define a flavor that has appropriate specifications: 
+** If you have administrative access to the {rh-openstack} deployment:app-name: 
```

Comment:
Because there isn't a preview of this file, I can't tell if `deployment:app-name: ` renders as intended, so it might be worth checking locally.

Reply:
Good catch. That's just bad autocompletion from VS Code :( 

```suggestion
** If you have administrative access to the {rh-openstack} deployment:
```